tasks/bbb-html5.yml

```diff
@@ -25,6 +25,17 @@
   when: not ((((bbb_meteor | default({})).public | default({})).media | default({})).sipjsHackViaWs  | default({})) | bool
   notify: reload nginx
 
+- name: install web.nginx
+  become: true
+  template:
+    src: nginx/web.nginx.j2
+    dest: /etc/bigbluebutton/nginx/web
+    owner: bigbluebutton
+    group: bigbluebutton
+    mode: 755
```

Comment:
I would prefer `mode: "0644"`.

Reply:
Default ist 755

Reply:
@BrutalBirdie is right. This unquotet mode-setting is wrong. It should be at least `mode: '755'`. But why should a config file need to be executable? Default for regular files is 644.